predict.py

```diff
@@ -191,14 +201,25 @@ async def predict(
             stop_sequences=args["stop_words"]
         )
 
+        start_time = time.time()
+        n_tokens = 0
+        tokens = np.array([], dtype=np.int32)
+
         async with req as resp:
             async for event in receive_sse(resp):
                 # Output is the _entire_ sequence, from the beginning
                 try:
-                    output = event.json()["text_output"]
+                    token = event.json()["output_ids"]
                 # this check can be removed once we identify the cause of KeyError
                 except Exception as e:
                     raise Exception(f"error with event {event}") from e
+                
+                n_tokens += 1
+                if n_tokens == 1:
+                    first_token_time = time.time()
+                
+                tokens = np.append(tokens, token)
+                output = self.tokenizer.decode(tokens, skip_special_tokens=True)
```

Comment:
this is running the whole output through the decoder for every prompt, right? anything to watch out for with very long outputs?